apps/next-evans/pages/friend/[friend_slug].tsx

```diff
@@ -1,36 +1,32 @@
 import { PrismaClient } from '@prisma/client';
+import invariant from 'tiny-invariant';
 import type { GetStaticPaths, GetStaticProps } from 'next';
-import type { Lang } from '@/../../libs-ts/types/src';
 import FeaturedQuoteBlock from '@/components/pages/friend/FeaturedQuoteBlock';
 import FriendBlock from '@/components/pages/friend/FriendBlock';
 import TestimonialsBlock from '@/components/pages/friend/TestimonialsBlock';
+import { LANG } from '@/lib/env';
+
+const client = new PrismaClient();
```

Comment:
i flip flopped on this one, i was getting warnings from prisma during deployment, it didn't seem to like having multiple instances of itself running